lib/vmpooler/pool_manager.rb

```diff
@@ -569,23 +574,38 @@ def migrate_vm_and_record_timing(vm_object, vm_name, pool, host, source_host_nam
       finish
     end
 
-    def check_pool(pool, maxloop = 0, loop_delay = 5)
-      $logger.log('d', "[*] [#{pool['name']}] starting worker thread")
+    def evaluate_pool(pool, slots, maxloop = 0, loop_delay = 1)
+      return if $redis.hget("vmpooler__pool__#{pool['name']}", 'slot')
+      checking_pools = $redis.smembers('vmpooler__check__pool__pending')
+      return if checking_pools.include? pool['name']
+      $redis.sadd('vmpooler__check__pool__pending', pool['name'])
+      loop_count = 1
+      while (slots_available?($redis.smembers('vmpooler__check__pool').count, slots.to_i).nil?)
+        sleep(loop_delay * 3)
```

Comment:
Should this be `* 3` ? The variable name kind of implies that it should it should delay for that many seconds, not three times that many seconds.